corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -528,7 +531,11 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
             e.preventDefault();
 
             self.validateAllFields().done(function () {
-                FormplayerFrontend.trigger("menu:query", self.getAnswers(), self.selectValuesByKeys);
+                FormplayerFrontend.trigger(
+                    "menu:query",
+                    self.getAnswers(),
+                    self.selectValuesByKeys,
+                    self.forceManualSearch);
```

Comment:
with stickler gone my nit is to have a comma and new line for the `)`

Reply:
I think the comma after a list of args is actually invalid syntax (or my linter thinks so), but happy to use:
```suggestion
                    self.forceManualSearch
                );
```